docs/assets/css/styles.scss

```diff
@@ -103,13 +104,30 @@ img {
 }
 
 pre>code {
-    background-color: #f5f5f5;
     display: block;
     padding: 20px;
     border-radius: 5px;
     overflow-x: auto;
 }
 
+// fenced codeblocks for plaintext
+.language-text,
+.language-plaintext {
+    pre>code {
+        // generic text from pygments theme
+        color: #d0d0d0;
+    }
+}
+
+// rogue syntax highlighting blocks for plaintext
+figure.highlight>pre>code {
+    &.language-text,
+    &.language-plaintext {
+        // generic text from pygments theme
+        color: #d0d0d0;
+    }
+}
```

Comment:
Because plaintext wasn't being themed properly automatically, even though it is a [supported language](https://github.com/rouge-ruby/rouge/wiki/List-of-supported-languages-and-lexers).